src/cmd/status.cpp

```diff
@@ -2,8 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#include <iostream>
 #include "status.h"
+
+#include <iostream>
```

Comment:
Can we please disable auto-code formatting? :)

Reply:
The formatter just replaced tabs with spaces and swapped a couple of includes.